src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -53,13 +54,20 @@ html {
             box-shadow: 2px 2px #7a766d;
             display: inline-block;
             padding: 1em;
-            min-height: 9.6em;
-            width: 25em;
+            min-width: 25em;
             background-color: #fff;
-            margin-right: 10px;
             white-space: normal;
-            vertical-align: middle;
+            display: table-cell;
         }
+        
+        .block {
```

Comment:
this should be in an fl-prefsEditor- namespace. It would also be good to know what it is used for. Maybe a comment or something. Also, what are the -webkit- prefixed styles for, why not just `display: block`?